docs/guide/catalog/catalog.md

```diff
@@ -35,3 +35,14 @@ An example of an attribute group can be dimensions (length, width, height).
 
 You can assign both whole attribute groups or individual attributes to a product type.
 
+## Product availability and stock
+
+Product availability defines whether a product is available in the catalog.
+
+!!! note
+
+    A product having availability does not automatically mean that it is in stock a possible to be ordered.
+    A product can be available, but have zero stock.
```

Comment:
@mikadamczyk has to confirm this, especially since we intend to make or already made changes to this.